src/shipit_uplift/shipit_uplift/coverage.py

```diff
@@ -260,14 +260,14 @@ def get_pushes_changesets(push_id, push_id_end):
 
 
 def get_changeset_data(changeset):
-    if changeset not in changeset_cache:
+    if changeset[:12] not in changeset_cache:
         r = requests.get('https://hg.mozilla.org/mozilla-central/json-rev/%s' % changeset)
         rev = r.json()
         push_id = int(rev['pushid'])
 
         get_pushes(push_id)
 
-    return changeset_cache[changeset]
+    return changeset_cache[changeset[:12]]
```

Comment:
The changeset hash is truncated in [`get_pushes`](https://github.com/gabriel-v/services/blob/313d7d5bb716b08fed5c44d93a6e1d551d8e9c0c/src/shipit_uplift/shipit_uplift/coverage.py#L240) but not here

Reply:
Good catch. It was harmless because the changeset is truncated in the callers, but better to be safe than sorry.